frontend/src/features/admin-form/responses/FeedbackPage/FeedbackTable.tsx

```diff
@@ -0,0 +1,149 @@
+import React, { useEffect } from 'react'
+import { Column, usePagination, useSortBy, useTable } from 'react-table'
+import { Box, Icon, Table, Tbody, Td, Th, Thead, Tr } from '@chakra-ui/react'
+
+import { ProcessedFeedbackMeta } from '~shared/types/form'
+
+import { BxsChevronDown } from '~assets/icons/BxsChevronDown'
+import { BxsChevronUp } from '~assets/icons/BxsChevronUp'
+
+type Data = {
+  index: number
+  date: string
+  feedback: string
+  rating: number
+}
+
+export const FeedbackTable = ({
+  feedbackData,
+  currentPage,
+}: {
+  feedbackData: ProcessedFeedbackMeta[] | undefined
+  currentPage: number
+}) => {
+  const columns = React.useMemo<Column<Data>[]>(
+    () => [
+      {
+        Header: '#',
+        accessor: 'index',
+        sortType: 'basic',
+        width: '5rem',
+      },
+      {
+        Header: 'Date',
+        accessor: 'date',
+        sortType: (rowA, rowB) => {
+          const dateA = new Date(rowA.values.id)
+          const dateB = new Date(rowB.values.id)
+          return dateA > dateB ? 1 : -1
+        },
```

Comment:
Is values.id a date string? What does it look like?

Also suggest extracting this column constant to outside of the component instead since we do not need any dependent data and it is truly a constant.

Reply:
yes, it is a date string and added documentation (actually it should be `values[id]` and  where `id` is a parameter for columnId, i've fixed it)

also moves constant out of component